templates/taxi/car_detail.html

```diff
@@ -2,6 +2,8 @@
 
 {% block content %}
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
+  <button><a href="{% url "taxi:car-update" pk=car.id %}">Update</a></button>
+  <button><a href="{% url "taxi:car-delete" pk=car.id %}">Delete</a></button>
```

Comment:
Using `<button>` elements to wrap `<a>` tags is not semantically correct and can lead to accessibility issues. Consider using CSS to style the `<a>` tags as buttons instead.